### PR TITLE
Add MANTRA Mainnet and Testnet images and chain registrations

### PR DESCRIPTION
**Pull request by mantramatt**
feat: add MANTRA Dukong Testnet and  MANTRA-1 Mainnet registrations and logos

### CODE REVIEW

**Comment by kamal-sutra** [comment]
@mantramatt Please check the failed validations. I think the image url's are incorrect.

**Comment by mantramatt** [comment]
Hi @kamal-sutra thanks for reviewing this for us. The images are actually in this PR. Do we need to host them externally on our CDN or is the preference to have them in this repository?

**Comment by kamal-sutra** [comment]
@mantramatt the image url is incorrect. please check the Details in the check which is not successful above.

File: cosmos/mantra-dukong.json [hunk 1]
```diff
@@ -0,0 +1,56 @@
+{
+    "chainId": "mantra-dukong-1",
+    "chainName": "MANTRA Dukong Testnet",
+    "chainSymbolImageUrl": "https://raw.githubusercontent.com/chainapsis/keplr-chain-registry/main/images/mantra/chain.png",
```
Comment (blacktoast) [inline cosmos/mantra-dukong.json:4]: You need an image for each chain identifier
For `mantra-dukong`, please also put the image in `images/mantra-dukong` and change the url to ‘https://raw.githubusercontent.com/chainapsis/keplr-chain-registry/main/images/mantra-dukong/chain.png’

Comment (mantramatt) [reply cosmos/mantra-dukong.json]: thanks @blacktoast, that is done.

**Pull request merged**